woocommerce/woocommerce-android
# Support billing address form in Order Creation. Save forms state in ViewModel.

Closes: #5289


### Description


This PR enables filling shipping address different than billing one.

Under-the-hood this PR also changes way we keep state of input fields. From now on we'll keep all state in one place, ViewModel. For now it only works for Order Creation/Add customer form but I do plan to adjust `BaseAddressEditingFragment` as well.

### Testing instructions


TC1 Regression tests
Please check any of edit order screens to make sure it still works.

TC2 Test `Add customer` screen in Create Order.
I intentionally don't add specific steps because the feature should feel natural to work with. If you see some unexpected behaviour, please share concern.

### Images/gif



https://user-images.githubusercontent.com/5845095/147233844-4e65f7d0-0ee3-4694-aef3-bad96df53cab.mov
